### ISSUE TITLE
[Question] Required module not found

### ISSUE THREAD

**Issue by raduchiriac** [bigslycat/flow-geojson#1]
EDIT: I am using yarn workspaces and lerna
Probably not a real issue but I am getting `flow-geojson. Required module not found` when using this.

```javascript
import type {
  Point2D,
  MultiPoint2D,
  LineString2D,
  MultiLineString2D,
  Polygon2D,
  MultiPolygon2D,
} from 'flow-geojson';
```

Following this did not help:
https://github.com/facebook/flow/issues/2092#issuecomment-232917073

Because I now get 
`Type annotations require valid Flow declaration  flowtype/no-types-missing-file-annotation`

**Comment by bigslycat** [comment]
Thanks for report. I review it in next 4 hours.

**Comment by bigslycat** [comment]
@raduchiriac thanks for report, but it just works. =)

![2017-12-16 00-01-08](https://user-images.githubusercontent.com/1121997/34060403-95fb4848-e1f4-11e7-832c-b46810580a6b.png)

Please read about this Flow ESLint plugin rule — [no-types-missing-file-annotation](https://github.com/gajus/eslint-plugin-flowtype#no-types-missing-file-annotation). Make sure you have the valid Flow file declaration.

Please tell me about result.

**Comment by raduchiriac** [comment]
It needed more clarifications from my side as I am actually using yarn workspaces with lerna. (I will edit the original question as well).

I first tried to `yarn add ` inside the subfolder where I needed this package, and since the package will be installed in the root, I am guessing the symlinks are not properly created and I got `flow-geojson. Required module not found`. Not sure how to debug this case.

The only workaround I found is doing `yarn add -W flow-geojson` in the root (which is not ideal, I only have jest, eslint, babel in the root) and removing `node_modules` from my .flowconfig ignore.

Let's leave it closed as it's a very specific case, will come back with more details
